app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt

```diff
@@ -15,18 +20,55 @@ import org.systers.mentorship.viewmodels.LoginViewModel
 /**
  * This activity will let the user to login using username/email and password.
  */
+
+private const val TAG = "LoginActivity"
+private const val RC_SAVE = 1
+private const val RC_READ = 2
+
 class LoginActivity : BaseActivity() {
 
     private lateinit var loginViewModel: LoginViewModel
 
     private lateinit var username: String
     private lateinit var password: String
 
+    private lateinit var mCredentialsClient: CredentialsClient
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
-
         loginViewModel = ViewModelProviders.of(this).get(LoginViewModel::class.java)
+
+        mCredentialsClient = Credentials.getClient(this)
+        val mCredentialRequest = CredentialRequest.Builder()
+                .setPasswordLoginSupported(true)
+                .build()
+        mCredentialsClient.request(mCredentialRequest).addOnCompleteListener {task -> run {
+            if (task.isSuccessful)
+            {
+                onCredentialRetrieved(task.result!!.credential)
+            }
+            else
+            {
+                val exception = task.exception
+                if (exception is ResolvableApiException)
+                {
+                    try
+                    {
+                        exception.startResolutionForResult(this, RC_READ)
+                    }
+                    catch (e: IntentSender.SendIntentException)
+                    {
+                        Log.e(TAG, "Failed to send resolution.", e)
+                        Toast.makeText(this, "Credential Read Failed", Toast.LENGTH_SHORT).show()
```

Comment:
Move this string to strings.xml and any other strings that follow

Reply:
okay